### PR TITLE
 fix broken sharing services (part 2): delete Blogotext

### PR DESCRIPTION
**Pull request by math-GH**
Closes a part of #3793

sub PR of #3794 as @Alkarex suggested

Changes proposed in this pull request:
- remove: the blogotext sharing option

How to test the feature manually:
check the sharing options

Pull request checklist:
- [x] clear commit messages
- [x] code manually tested

### CODE REVIEW

**Comment by math-GH** [comment]
What is the plan with this PR?
Set it on hold?
Merge it and wait if blogotext is still needed?

File: app/shares.php
Comment (Frenzie) [inline app/shares.php]: I don't know anything about blogotext, but I will make the semantic point that unmaintained for a little while isn't the same thing as broken. This help link is broken though; presumably that should be something like https://github.com/BlogoText/blogotext instead.

So I'm on the side of keep. It's not like something with fixes for PHP 7.3/8.0 is something hugely outdated that'll only run on 5+ year old systems, even if it might end up there a few years down the line.

Comment (math-GH) [reply app/shares.php]: feel free to refuse this PR.

feel free to fix the link.

As I feel free to delete old stuff that does not make sense to maintain. Blogotext creates 2 PR where blogotext code had to change (1 PR that reordered the array, 1 PR that will fix the link). I do not like technical debts.

If we do not want to delete it I would suggest to mark blogotext as deprecated with the hint that users of FreshRSS and blogotext should sent a message to this issue to mark it as "still in use".

Comment (Alkarex) [reply app/shares.php]: One of the issues is that we have, by design, no telemetry for which features are used, and very little feedback from our user-base, so I tend to be conservative and wait a bit more before deleting things. I can see some user activity from April this year on https://github.com/BlogoText/blogotext/issues
Having a deprecated indication could be interesting.

**Comment by math-GH** [comment]
Blogotext as sharing service in FreshRSS will be marked as deprecated in 1.20.0 and is planned to be removed in 1.22.0 (see #4269)

**Comment by math-GH** [comment]
@Alkarex Is there any reason for postponing this PR from 1.22 to 1.23?

**Comment by Alkarex** [comment]
> @Alkarex Is there any reason for postponing this PR from 1.22 to 1.23?

@math-GH It was mostly to wait a bit more, with the same reasoning as https://github.com/FreshRSS/FreshRSS/pull/3812#discussion_r702465903 . I would rather merge at the start of a new development phase, to get some time to hear whether anyone complains

**Comment by math-GH** [comment]
> > @Alkarex Is there any reason for postponing this PR from 1.22 to 1.23?
> 
> @math-GH It was mostly to wait a bit more, with the same reasoning as [#3812 (comment)](https://github.com/FreshRSS/FreshRSS/pull/3812#discussion_r702465903) . I would rather merge at the start of a new development phase, to get some time to hear whether anyone complains

Is now a good time?

**Review by Frenzie — APPROVED** [review]
We're now a few years down the line.

**Comment by Alkarex** [comment]
Sorry I forgot about it. Could be merge right after the release of 1.24?

**Comment by math-GH** [comment]
see #6225

**Pull request closed without merging**
